gapic/src/main/com/google/gapid/models/Settings.java

```diff
@@ -220,6 +220,10 @@ public void save() {
     return proto.getPerfettoBuilder();
   }
 
+  public SettingsProto.DeviceValidation.Builder deviceValidation() {
```

Comment:
follow the pattern of using `writeFoo` if you are returning a builder, to indicate that this can mutate the proto.